bitcoin/script/script.h

```diff
@@ -578,12 +578,12 @@ template<typename... Ts>
 CScript BuildScript(Ts&&... inputs)
 {
     CScript ret;
-    size_t cnt{0};
+    int cnt{0};
```

Comment:
Using a signed type may be better here, because the compiler knows those cannot overflow. `int` should be sufficiently large in any case.